.github/workflows/build.yml

```diff
@@ -73,17 +73,15 @@ jobs:
             export CC=clang
             export DISTCHECK=true
 
-            # Do what USES="localbase:ldflags gettext-runtime gmake" do
+            # Do what USES="localbase:ldflags gmake" do
             # in FreeBSD Ports, namely:
             # - Add /usr/local/include to the compiler's search path
             # - Add /usr/local/lib to the linker's search path
-            # - Additionally link to libintl because it is a separate library,
-            #   unlike on Linux. See https://github.com/avahi/avahi/issues/726
             # - Use GNU make instead of BSD one
             export MAKE="gmake"
             export CFLAGS="-I/usr/local/include"
             export CPPFLAGS="-I/usr/local/include"
-            export LDFLAGS="-L/usr/local/lib -lintl"
+            export LDFLAGS="-L/usr/local/lib"
```

Comment:
I moved all this stuff to `build.sh` in https://github.com/avahi/avahi/pull/740 so `-lintl` should be removed there instead.